docs/quick-start.md

```diff
@@ -0,0 +1,214 @@
+---
+sidebar_label: Quick Start
+sidebar_position: 2
+---
+
+# Quick Start
+
+This page will guide you through downloading TigerBeetle, setting up a single-
+or multi-node cluster, and creating some accounts and transfers using the REPL.
+
+## 1. Download TigerBeetle
+
+TigerBeetle is a single, small, statically-linked binary.
+
+### Bootstrap Script
+
+#### macOS / Linux
+
+```shell
+git clone https://github.com/tigerbeetle/tigerbeetle && cd tigerbeetle && ./bootstrap.sh
+```
+
+#### Windows
+
+```powershell
+git clone https://github.com/tigerbeetle/tigerbeetle && cd tigerbeetle && .\bootstrap.ps1
+```
+
+#### Build from Source
+
+Add `-build` as an argument to the bootstrap script to build from source.
+
+#### Direct Download
+
+Instead of using the bootstrap script, you can download prebuilt binaries here:
+
+|         | Linux                           | Windows                          | MacOS                             |
+| :------ | :------------------------------ | :------------------------------- | :-------------------------------- |
+| x86_64  | [tigerbeetle-x86_64-linux.zip]  | [tigerbeetle-x86_64-windows.zip] | [tigerbeetle-universal-macos.zip] |
+| aarch64 | [tigerbeetle-aarch64-linux.zip] | N/A                              | [tigerbeetle-universal-macos.zip] |
+
+[tigerbeetle-aarch64-linux.zip]:
+  https://github.com/tigerbeetle/tigerbeetle/releases/latest/download/tigerbeetle-aarch64-linux.zip
+[tigerbeetle-universal-macos.zip]:
+  https://github.com/tigerbeetle/tigerbeetle/releases/latest/download/tigerbeetle-universal-macos.zip
+[tigerbeetle-x86_64-linux.zip]:
+  https://github.com/tigerbeetle/tigerbeetle/releases/latest/download/tigerbeetle-x86_64-linux.zip
+[tigerbeetle-x86_64-windows.zip]:
+  https://github.com/tigerbeetle/tigerbeetle/releases/latest/download/tigerbeetle-x86_64-windows.zip
+
+#### Docker
+
+You can find instructions on using TigerBeetle with Docker
+[here](./deploy/docker.md).
```

Comment:
I think we can safely cut docker from quickstart completely, and instead linkg the overall deploy section at the end

Reply:
My guiding principle here would be that quick start should include _one_ blessed way of doing things, and, if there are multiple options, those should be provided not in the quick start, but rather in a topical guide.

So, we probably should have "installation" page somewhere in operations section, where we describe supported platforms, how to build from source, how to verify checksums, etc. Installing via docker would fit there. 

In the quick start, we can then link that page saying something like "see alternative installation instructions there"

Reply:
And, for docker specifically, as we want to de-emphasize it anyway, I think it's fine if its only a part of "Docker" topic, and not mentioned in the installation.

But no strong opinion here!

Reply:
Maybe we just go back to having a top-level Download page that has all of the various instructions. We can leave only the bootstrap script option in the quickstart

Reply:
:thinking: I don't think this needs to be a top-level option, it feels very much like a part of ops. And long term we definitely should have a dedicated installation/download page.

_But_ I feel that this actually strays off-topic for this PR, perhaps better to leave that to a follow up, once we do #1896 which would require changing installation instructions anyway. 

Happy to approve with or without docker, this is a marked improvement already! But CI needs to be green!